bin/local-env/env-check.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Include useful functions
+. "$(dirname "$0")/includes.sh"
+
+# Check for the site status
+if [[ -z $(curl -s --head --fail $npm_package_config_E2E_URL) ]]; then
+	echo -e $(status_message 'e2e environment is not running, starting it now.')
+	npm run env:start
+	exit 0
```

Comment:
These `exit` statements are unnecessary, the script will `exit 0` automatically if there is no error.

We should however, add a `set -e` at the top to make sure the script exits with an error if `env:start` fails.